client/python/cli/command/profiles.py

```diff
@@ -72,11 +74,15 @@ def _create_profile(self, name: str) -> None:
             client_secret = input("Polaris Client Secret: ")
             host = input(f"Polaris Host [{DEFAULT_HOSTNAME}]: ") or DEFAULT_HOSTNAME
             port = input(f"Polaris Port [{DEFAULT_PORT}]: ") or DEFAULT_PORT
+            realm = input(f"Polaris Context Realm: ")
+            header = input(f"Polaris Context Header Name [{DEFAULT_HEADER}]: ") or DEFAULT_HEADER
             profiles[name] = {
                 "client_id": client_id,
                 "client_secret": client_secret,
                 "host": host,
                 "port": port,
+                Arguments.REALM: realm,
```

Comment:
Just curious (nit): why using a different syntax here ? Why not using directly `realm` ?